lib/utils/parseRequest.js

```diff
@@ -1,3 +1,17 @@
+
+
 module.exports = rawRequest => {
+  const parsedRequest = rawRequest.split('\r\n');
+  const [method, path] = parsedRequest[0].split(' ');
+
+  if(parsedRequest[7]){
+    const body = parsedRequest[7];
+    return ({ method, path, body });
+  }
 
+  if(parsedRequest[2]) {
+    const body = parsedRequest[2];
+    return ({ method, path, body });
```

Comment:
For this line and line 16, you can omit the parenthesis:

```suggestion
    return { method, path, body };
```